docs/03-github/02-activation.mdx

```diff
@@ -60,6 +60,31 @@ Commit and push your workflow definition.
 
 #### Converting into a license
 
+:::caution Unity no longer supports manual activation of Personal licenses
```

Comment:
Too long for a caution title; looks messy on mobile.

Recommend either simply using no title in which case you'll see `⚠️ CAUTION` as the title, by moving the title down into the regular text. Or to come up with a slightly shorter version of this title.

Reply:
I would keep the `Unity no longer supports manual activation of Personal licenses` message as this is the actual message we see on Unity's website as shown on the activation page:

![image](https://github.com/game-ci/documentation/assets/1264761/d7724973-1fec-4e4c-a47e-6a2aa9879679)

I can move it down the title and make it bold instead.

Before:

![ScreenShot CleanShot 2023-09-06 at 21 34 12](https://github.com/game-ci/documentation/assets/1264761/8c571053-9dc5-4472-bcc1-d52d9cb0807d)

after:

![ScreenShot CleanShot 2023-09-06 at 21 35 06](https://github.com/game-ci/documentation/assets/1264761/027509ab-7915-429e-ae86-8f2848bc8388)